### ISSUE TITLE
Provide more options to dosbox config

### ISSUE THREAD

**Issue by caiiiycuk** [caiiiycuk/js-dos.com#75]
```
[dosbox]
#  machine: The type of machine tries to emulate.
#           Possible values: hercules, cga, tandy, pcjr, ega, vgaonly, svga_s3, svga_et3000, svga_et4000, svga_paradise, vesa_nolfb, vesa_oldvbe.
#  memsize: Amount of memory DOSBox has in megabytes.
#             This value is best left at its default to avoid problems with some games,
#             though few games might require a higher value.
#             There is generally no speed advantage when raising this value.

machine=svga_s3
memsize=16
```

```
[autoexec]
# Lines in this section will be run at startup.
# You can put your MOUNT lines here.
```

add clear **OR** into js-dos.com

**Comment by caiiiycuk** [comment]
Fixed in 6.22.31

**Comment by caiiiycuk** [comment]
*6.22.41
